src/h_vialib/secure/nonce.py

```diff
@@ -0,0 +1,29 @@
+"""Create values we can check we created."""
+
+from uuid import uuid4
+
+from h_vialib.secure.token import SecureToken
+
+
+class RandomSecureNonce(SecureToken):
+    """A random token we can verify we created with an expiry time.
+
+    This token should be different every time you create it.
+    """
+
+    def create(self, expires=None, max_age=None):  # pylint: disable=arguments-differ
+        """Create a random nonce which changes every time."""
+        return super().create(
+            payload={"salt": uuid4().hex}, expires=expires, max_age=max_age
```

Comment:
In checkmate we used a SHA hashing thing with hex salt... but this is so much easier.